code/mob/living/critter/ai/wraith_critters/exploder.dm

```diff
@@ -0,0 +1,76 @@
+//Todo, fix ai tick rate. Seems to stop to think too often. Also stops before attacking.
+
+/datum/aiHolder/wraith_critters/exploder
+
+/datum/aiHolder/wraith_critters/exploder/New()
+	..()
+	default_task = get_instance(/datum/aiTask/prioritizer/exploder, list(src))
```

Comment:
the reason it's ticking slow is because we fixed some things in flock that are not pushed upstream yet that does faster task switching. You could also try just replacing the `aiTask/prioritizer` with your `goalbased/rushdown` task instead, if you don't mind the exploder not wandering when there are no targets.

Reply:
I might do so for now for testing purposes, while waiting for the new flock ai things!